integration-tests/discovery-tests/order-consumer/src/main/resources/bootstrap.yaml

```diff
@@ -48,6 +48,10 @@ spring:
       swagger:
         enableJavaChassisAdapter: true
 
+    webmvc:
+      publicKey:
+        consumerEnabled: true
+
```

Comment:
Why need add this? Test cases for servicecomb should run.

Reply:
fixed